scripts/build_country_flh.py

```diff
@@ -198,7 +174,7 @@ def build_aggregate(flh, countries, areamatrix, breaks, p_area, fn):
     # Use GLAES to compute available potentials and the transition matrix
     paths = dict(snakemake.input)
 
-    init_globals(bounds, dx, dy, config, paths)
+    init_globals(bounds.xXyY, dx, dy, config, paths)
```

Comment:
Or this one?

Reply:
as well as this, but it is necessary for running the script. I can however create an issue for that